SwiftReflector/NewClassCompiler.cs

```diff
@@ -148,7 +149,8 @@ public class NewClassCompiler {
 
 		public NewClassCompiler (SwiftCompilerLocation swiftCompilerLocations, ClassCompilerOptions options, UnicodeMapper unicodeMapper)
 		{
-			SwiftCompilerLocations = SwiftRuntimeLibrary.Exceptions.ThrowOnNull (swiftCompilerLocations, nameof (swiftCompilerLocations));
+			ReflectorLocations = SwiftRuntimeLibrary.Exceptions.ThrowOnNull (swiftCompilerLocations, nameof (swiftCompilerLocations));
+			SwiftCompilerLocations = new SwiftCompilerLocation ("/usr/bin", "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-5.0/macosx");
```

Comment:
Any possible way to get the path from the makefile? That way if we bump it, you wont have to change this.